applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -246,14 +238,11 @@ void RegisterCurveGeometry(
     pybind11::module& m,
     const std::string& name)
 {
-    namespace py = pybind11;
     using namespace pybind11::literals;
 
-    using VectorType = Kratos::array_1d<double, TDimension>;
-
-    using Type = ANurbs::CurveGeometry<VectorType>;
-    using Holder = ANurbs::Pointer<Type>;
-    using Base = ANurbs::CurveGeometryBase<VectorType>;
+    using Type = CurveGeometry<TDimension>;
+    using Holder = shared_ptr<Type>;
```

Comment:
what is this? an `std` or a `Kratos`  `shared_ptr`?
=> can you please prepend `Kratos::`? (also in other places)

Reply:
Well I am already in the Kratos namespace... but with the shared_ptr it might make sense to explicit add the namespace. Otherwise some people might be confused.

Reply:
yes it would be good to add it IMO